ftw/solr/browser/livesearch.py

```diff
@@ -210,6 +228,41 @@ def get_nothing_found_item(self):
         return {
             'url': url,
             'title': translate(label_nothing_found, context=self.request),
+            'description': translate(label_suggestion, context=self.request),
             'first_of_group': False,
-            'cssclass': 'no-result'
+            'cssclass': 'no-result',
         }
+
+    def suggestions(self):
+        """Get suggestions from spellcheck component.
+        This a copy of the search.py suggestions method.
+        """
+
+        if not safe_hasattr(self, 'solr_response'):
+            return []
+
+        suggested_terms = []
+        search_terms = [term.lower() for term in self.request.get(
+            'term', '').split()]
+        query_params = self.request.form.copy()
+
+        if safe_hasattr(self.solr_response, 'spellcheck'):
+            suggestions = self.solr_response.spellcheck.get('suggestions', [])
+            for term in search_terms:
+
+                if not isinstance(term, unicode):
+                    term = term.decode('utf-8')
+
+                if term in suggestions:
+                    suggestion = suggestions[term]['suggestion']
+                    query_params['SearchableText'] = suggestion[0]['word']
+                    query_string = ''
+                    for k, v in query_params.items():
+                        if isinstance(v, list):
+                            query_string += '&' + '&'.join(
+                                ['%s=%s' % (k, vv) for vv in v])
```

Comment:
Are those values escaped correctly?
I would prefer using [`urllib.urlencode`](https://docs.python.org/2/library/urllib.html#urllib.urlencode) so that all the edge cases are supported.